app/models/redirect.rb

```diff
@@ -0,0 +1,20 @@
+class Redirect
+  DEFAULT = ''.freeze
+  attr_reader :path
+
+  def initialize(path)
+    @path = path
+  end
+
+  def title
+    @title ||= begin
+      Services.content_store.content_item(@path).to_h['title'] || DEFAULT
```

Comment:
Do you need `.to_h` here? Could you instead do `.fetch("title", DEFAULT)` perhaps?

Reply:
The thing is that the hash can contain ```{title: nil}``` and so this that would return nil. Otherwise I'd have to use ```.fetch('title', DEFAULT) || DEFAULT``` which isn't great.